README.md

```diff
@@ -31,16 +34,7 @@ or
 
     $ python3 --version
 
-Next, install [Poetry](https://python-poetry.org/) by using the [official
```

Comment:
Won't we still need some of this information for anyone developing the tool? My answer is "yes" but these changes indicate you say "no." We could move some content to a `DEVELOPER.md` file if the `README` needs to be lean.